### ISSUE TITLE
loading in progress, circular require considered harmful - /home/public/Dropbox/common/ruby/lib/break.rb

### ISSUE THREAD

**Issue by zw963** [gsamokovarov/break#8]
When work with ruby 3.0.0, raise following warn.

```
<internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85: warning: <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85: warning: loading in progress, circular require considered harmful - /home/public/Dropbox/common/ruby/lib/break.rb
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from /home/public/Dropbox/common/ruby/lib/debugrc.rb:7:in  `<top (required)>'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from /home/public/Dropbox/common/ruby/pry/pry_hack.rb:11:in  `<top (required)>'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from /home/public/Dropbox/common/ruby/lib/break.rb:12:in  `<top (required)>'
	from /home/public/Dropbox/common/ruby/lib/break.rb:12:in  `require_relative'
	from /home/public/Dropbox/common/ruby/lib/break/pry.rb:5:in  `<top (required)>'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
	from <internal:/home/zw963/others/.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in  `require'
```

**Comment by gsamokovarov** [comment]
Should be fixed in `0.30.0`. Sorry for taking so long to respond and feel free to reopen the issue if the problem persists.

**Comment by zw963** [comment]
it works, thank you!
